models/articles.models.js

```diff
@@ -2,10 +2,35 @@ const db = require("../db/connection");
 
 exports.readArticleById = (article_id) => {
   return db
-    .query(`SELECT * FROM articles WHERE article_id=$1`, [article_id])
+    .query(
+      `
+      ALTER TABLE articles
+      ADD comment_count INT 
```

Comment:
I see that it works but this looks messy to me. Have a think about how you can achieve the same result with much less code. It can be all done in 1 db query. You can do this by using an aggregate function to "count" the number of entries there are in the comments table and using that to add the comment count to your articles table 🙂